### PR TITLE
Initial work towards mac support

### PR DESCRIPTION
**Pull request by pokey**

### CODE REVIEW

File: lib/record_data.py [hunk 1]
```diff
@@ -48,9 +49,8 @@ def record_controls( recordQueue=None ):
                 ## If the audio queue exists - make sure to clear it continuously
                 if( recordQueue != None ):
                     recordQueue.queue.clear()
-
```
Comment (pokey) [inline lib/record_data.py:51]: @chaosparrot any reason we couldn't put a sleep in this loop so it doesn't spin up my cpu?  Like 50ms?

Comment (chaosparrot) [reply lib/record_data.py:51]: Sounds good!

**Comment by chaosparrot** [comment]
Any chance we could change the destination branch to feature/crossplatform? I want to add some documentation about the usage on Mac ( i.e. the installation process, setting up the permissions ) before merging to master.

**Comment by pokey** [comment]
> feature/crossplatform

Done: #4

**Pull request closed without merging**